docs: Small fix and update of README.md (#740)
awslabs/syne-tune: docs/source/getting_started.rst

| :class:`~syne_tune.optimizer.schedulers.transfer_learning.BoundingBox`                 | `Perrone, et al. (2019) <https://arxiv.org/abs/1909.12552>`__                   | any           | yes           | yes             | yes       |
+----------------------------------------------------------------------------------------+---------------------------------------------------------------------------------+---------------+---------------+-----------------+-----------+

:sup:`*`: We implement the model-based scheduling logic of DyHPO, but use the

> same Gaussian process surrogate models than MOBSTER or HyperTune. The original

Reviewer: DyHPO/MOBSTER/HyperTune all use the same GP surrogate models? If yes, then I'd rephrase as: 

```suggestion
same Gaussian process surrogate models as MOBSTER and HyperTune. The original
```
